documentation/manual/src/main/asciidoc/modules/mongodb.asciidoc

```diff
@@ -3473,6 +3473,55 @@ List<Poem> result = session.createNativeQuery( nativeQuery ).addEntity( Poem.cla
 ----
 ====
 
+[[ogm-mongodb-stored-proc-native]]
+
+==== Server-side JavaScript and stored procedures
+
+In MongoDB, it is possible to call server side javascripts as if it is a stored procedure.
+You can use the existing methods in JPA:
+
+.Calling server side javascript with Hibernate OGM with positional parameters
+====
+[source, JAVA]
+----
+  @Entity
+  public class Car {
+    @Id
+    private Integer id;
+
+    private String brand;
+
+    ...
+  }
+
+  EntityManager em = ...
+  StoredProcedureQuery storedProcedureQuery = em.createStoredProcedureQuery( "findMostExpensiveCars", Car.class );
+  storedProcedureQuery.registerStoredProcedureParameter( "year", Integer.class, ParameterMode.IN );
+  storedProcedureQuery.setParameter( "year", 1995 );
+  List<Car> cars = storedProcedureQuery.getResultList();
+----
+====
+
+This example will work assuming that there is a javascript function in MongoDB defined
+`findMostExpensiveCars` and that the result of the function is a list of cars that can be mapped
+by the entity `Car`.
+
+.Calling server side javascript with Hibernate OGM with positional parameters
+====
+[source, JSON]
+----
+{
+  "result" : [
+    { "id":1, "brand":"Bentley" },
+    { "id":2, "brand":"Maserati" },
+  ]
+}
+----
+====
+
+I will let the details about server side funtions to
+https://docs.mongodb.com/manual/core/server-side-javascript[the MongoDB reference documentation].
```

Comment:
Maybe "More details about server-side JavaScript functions can be found in the MongoDB reference documentation."

We usually avoid the `I` in documentation.